.nycrc.json

```diff
@@ -17,6 +17,8 @@
     "test-fixtures",
     "scripts",
     "coverage",
-    "build"
+    "build",
+    ".github",
+    ".next"
```

Comment:
Would it make sense to delete `.next` on your machine instead? It's not going to be there in recent clones.

Reply:
Ha! Good point